_episodes/01-intro.md

```diff
@@ -146,27 +145,27 @@ and then print the prompt and wait for you to enter another command.
 
 ### Is it difficult?
 
-It isn't difficult, but it is a different model of interacting than a GUI, and that 
-will take some effort - and some time - to learn. A GUI 
+Using the command shell is not difficult, but a different method of interacting with the computer than a GUI, and 
+will take some effort - and time - to learn. A GUI 
 presents you with choices and you select one. With a CLI the choices are combinations 
-of commands and parameters, more like words in a language than buttons on a screen. They
+of commands and parameters, more like words in a language than buttons on a screen. These choices
 are not presented to you so
-you must learn a few, like learning some vocabulary in a new language. But a small 
-number of commands gets you a long way, and we'll cover those essential few today.
+you must learn the commands and parameters, similar to learning the vocabulary in a new language. A small 
+number of commands will go a long way, and we will cover those that are essential today.
```

Comment:
I don't think the modifications are an improvement. The first sentence no longer makes sense.

Reply:
How about keep the first original sentence and keep the revised second one. The  "small number of commands gets you a long way" doesn't sound quite right.